apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ValueConversion.cs

```diff
@@ -44,6 +44,11 @@ public static object ConvertToClrType(this Value wireValue, V1.Type spannerType,
                 }
                 return null;
             }
+            if (targetClrType == typeof(object))
```

Comment:
you could pass in a bunch of base types that could 'theoretically' work, but its not always clear what they mean and if they should.  "ValueType" -- ok, *which* value type did you want?  IComparable?  That doesn't give us any what the result should be.

In most cases, you need to pass in a concrete, constructable type or value type (eg bool). 

typeof(object) as an indication of "I don't care" seems reasonable to me as that's what you'd do in a generic type in the same situation.  Want a dictionary of "string" to "I-don't-care-it-could-be-anything"?  Dictionary<string, object>